### ISSUE TITLE
Issue while running JAVA libzt_java example 

### ISSUE THREAD

**Issue by balajigite16** [zerotier/libzt#66]
Getting issue in zt.dll library, i tried adding dll in 
1. Adding in java path while running application
![image](https://user-images.githubusercontent.com/32266094/81301146-42a2e300-90ab-11ea-8bb4-fff316e68513.png)

2. Adding into  Environment variable path in system
3. Adding under C:\Program Files\Java\jdk1.8.0_231\jre\lib - file 

i tried build dll from vs -2017 zerotierone solution 

still i am getting issue running it could anyone knows the solution on it

**Comment by balajigite16** [comment]
Hi @joseph-henry 

I don't know whoom to talk to, I can't see my issue in issue list of zerotierOne issue list. Could anyone help me out on this regards.

BALAJI R GITE

**Comment by rossmc7** [comment]
@balajigite16  I'm having the same issue, working on a Mac rather than windows. 

From your screenshot I can see that you used the bundled .java files. I've checked the created .so file, and the init method does appear to be in the binary, however, when I checked which methods were publicly accessable in the library I could not see the init method, only about half of the methods. 

accept, bind, recv, write, read, send, select, sendto, shutdown, listen to list a few.

I'm thinking either the issue is with the JNI, or with the source itself, but I compiled the mac binary as well, and that seemed to work I think it's an issue with JNI.
